lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -317,8 +700,175 @@ def add_attr(name):
     add_attr("sensor_id")
     add_attr("meteosat_id")
     add_attr("alphas_available")
+    if "radar" not in field.source:
+        for key in [
+            "neighbourhood_radius",
+            "recursive_filter_iterations",
+            "recursive_filter_alpha",
+            "threshold_vicinity_radius",
+            "probability_period_of_event",
+        ]:
+            add_attr(key)
+
+    source = field.source.strip()
+    rematcher = re.compile(r"^ek\d\d$")
+    if rematcher.match(source) is not None or source.find("umek") == 0:
+        source = "MOGREPS-UK"
+    cube.attributes["source"] = source
+    cube.attributes["title"] = "Unknown"
+    cube.attributes["institution"] = "Met Office"
+
+
+def known_threshold_coord(field):
+    """
+    Supplies known threshold coord meta-data for known use cases.
+    """
+    coord_keys = {}
+    if field.field_code == 161 and field.threshold_value >= 0.0:
+        coord_keys = {"var_name": "threshold"}
+        if field.threshold_value_alt > 8.0:
+            coord_keys["standard_name"] = "height"
+            coord_keys["units"] = "metres"
+        else:
+            coord_keys["standard_name"] = "cloud_area_fraction"
+            coord_keys["units"] = "oktas"
+    if field.field_code == 29 and field.threshold_value >= 0.0:
```

Comment:
Given the way that 'coord_keys' is used, by simple assignment, I think it would be clearer if all the 'if's here were 'elif's.
If I understand right, these cases should all be mutually exclusive, we should never get a later one triggering and overwriting the results of a previous one ??

Reply:
Now `elif`s